source/developers/cook-books/blueprint/index.rst

```diff
@@ -213,10 +213,6 @@ where the following fields are required:
 
 - ``descriptorVersion`` - The version of the format for this file (You can copy the value from the ``craftercms-plugin.yaml`` in one of the default blueprints under ``CRAFTER_INSTALL/data/repos/global/blueprints/`` if using a copy from the default)
 - ``plugin.type`` - ``blueprint`` for our purposes
-- ``plugin.id`` - a unique Id that is meaningful/recognizable to people who will be using the blueprint/plugin
-- ``plugin.name`` - blueprint/plugin name (For our blueprint example, it is the blueprint name shown in the **Choose Blueprint** screen of **Create Site**)
-- ``plugin.version`` - a version number for the blueprint
```

Comment:
And we want this.

Reply:
Oops... Blindly removed what was shown and did not check what was highlighted. Will return the `plugin.id`, `plugin.name` and `plugin.version`